src/player.py

```diff
@@ -27,17 +27,17 @@ def add_to_trait(self, name, value):
         """Add 'value' to 'trait'"""
         pass
 
-    def add_items(self, items):
+    def change_items(self, items):
         """Add all items in 'items' (dictionary name to value) in the player's inventory"""
```

Comment:
- i think function name would make more sense as `update_items`
- maybe add type hints like `items: dict`
- docstring is phrased a bit weirdly. try something like: "Update the players inventory to reflect the parameter `items`."